job/nullable_int.go

```diff
@@ -0,0 +1,51 @@
+package job
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// NullableInt represents an int that can be "empty" or nil.
```

Comment:
I am not sure I undertand the

> empty **or** nil

Clause. AFAICT this is a datatype for handling JSON values that can be either `null` or an integer. On a side note, the usage of this is to use the default maxRetries as a fallback. To me it makes more sense to ommit the field altogether when you want to use the default

Reply:
It's actually  `"empty" (aka nil)`. I'll rephrase the comment to make it clearer.